src/_posts/databases/postgresql/2000-01-01-overview.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Scalingo for PostgreSQL® Overview
+nav: Overview
+modified_at: 2023-11-20 00:00:00
+tags: database postgresql addon
+index: 1
+---
+
+## PostgreSQL®
+
+PostgreSQL® is a free, open-source, community-managed object-relational
+database management system (ORDBMS) focusing on SQL compliance and
```

Comment:
suggestion: this acronym is not used anywhere else in the text. Hence I would mention it.
```suggestion
database management system focusing on SQL compliance and
```